import java.util.Scanner;
public class Pattern7{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int count = 0;
        for(int i=2; i<=n; i++){
            while(n%i == 0){
                
               count++; 
               n = n / i;
            }
           

        }
         if(count == 2){
                System.out.print("yes");
            }
            else {
                System.out.print("No");
            }
    }
}